x-pack/filebeat/module/cisco/amp/test/cisco_amp2.ndjson.log-expected.json

```diff
@@ -0,0 +1,5676 @@
+[
+    {
+        "@timestamp": "2021-01-15T11:59:52.000Z",
+        "cisco.amp.computer.active": true,
+        "cisco.amp.computer.connector_guid": "test_connector_guid",
+        "cisco.amp.computer.external_ip": "8.8.8.8",
+        "cisco.amp.computer.network_addresses": [
+            {
+                "ip": "10.10.10.10",
+                "mac": "87:c2:d9:a2:8c:74"
+            }
+        ],
+        "cisco.amp.connector_guid": "test_connector_guid",
+        "cisco.amp.event_type_id": 1107296344,
+        "cisco.amp.related.mac": [
+            "87:c2:d9:a2:8c:74"
+        ],
+        "cisco.amp.tactics": [
+            {
+                "description": "<p>The adversary is trying to avoid being detected.</p>\n\n<p>Defense Evasion consists of techniques that adversaries use to avoid detection throughout their compromise. Techniques used for defense evasion include uninstalling/disabling security software or obfuscating/encrypting data and scripts. Adversaries also leverage and abuse trusted processes to hide and masquerade their malware. Other tactics\u2019 techniques are cross-listed here when those techniques include the added benefit of subverting defenses.</p>\n",
+                "external_id": "TA0005",
+                "mitre_name": "tactic",
+                "mitre_url": "https://attack.mitre.org/tactics/TA0005",
+                "name": "Defense Evasion"
+            }
+        ],
+        "cisco.amp.techniques": [
+            {
+                "data_sources": "File monitoring, Process monitoring, Process command-line parameters",
+                "description": "<p>Adversaries may search local system sources, such as file systems or local databases, to find files of interest and sensitive data prior to Exfiltration.</p>\n\n<p>Adversaries may do this using a <a href=\"https://attack.mitre.org/techniques/T1059\">Command and Scripting Interpreter</a>, such as <a href=\"https://attack.mitre.org/software/S0106\">cmd</a>, which has functionality to interact with the file system to gather information. Some adversaries may also use <a href=\"https://attack.mitre.org/techniques/T1119\">Automated Collection</a> on the local system.</p>\n",
+                "external_id": "T1005",
+                "mitre_name": "technique",
+                "mitre_url": "https://attack.mitre.org/techniques/T1005",
+                "name": "Data from Local System",
+                "permissions": "",
```

Comment:
drop this?

Reply:
I believed I had my cleanup painless script I often used, but forgot to add it in, should be fixed now in the upcoming commit.